slidedeck/src/slides/07-forms/00-forms.pug

```diff
@@ -38,6 +38,20 @@ section
     <input type="text" value={this.state.city.value}
            onChange={this.handleCityChange} />
 
+section
+  h3 Forms are hard
+  ul
+    li Inherently working with user-supplied (dangerous!) data
+    li Need to pre-populate, auto-format, validate, marshal data
+    li Error handling is tricky
+    li Dynamic aspects are hard to manage manually
+
+section
+  h3 What do we do?
+  p You can do all of your forms manually if you want
+  p Highly recommend using a third-party library
+  p Lots of options, simple (basic) to complex (highly capable)
```

Comment:
All looks good. note for GVL, they are using Ant Design which has it's own form library.

It works like the others, though it doesn't require redux which is nice https://ant.design/components/form/